engine/include/cubos/engine/render/shadows/caster.hpp

```diff
@@ -0,0 +1,25 @@
+/// @file
+/// @brief Component @ref cubos::engine::ShadowCaster.
```

Comment:
```suggestion
/// @brief Struct @ref cubos::engine::ShadowCaster.
```

This isn't meant to be used as a component, right?